api/error_recorder.go

```diff
@@ -0,0 +1,33 @@
+package api
```

Comment:
Let's put them into api/debug.go as it's already part of debug API and we have ErrorRecord used in DebugLog there.